fleet/lib/rook-ceph-cluster/overlays/luan/values.yaml

```diff
@@ -1,33 +1,45 @@
 ---
 cephClusterSpec:
   mon:
-    count: 5
+    count: 3
```

Comment:
We probably want all nodes in a 5 node cluster to be a control-plane node and have a mon.